packages/lisk-transactions/src/constants.ts

```diff
@@ -40,6 +40,21 @@ export const BYTESIZES = {
 	DATA: 64,
 };
 
+interface FeeMap {
+	readonly [type: number]: number
+}
+
+export const FEES: FeeMap = {
```

Comment:
lets remove feemap and assign it each custom transaction since it's static fee?
https://github.com/LiskHQ/lisk-elements/blob/transaction_improvement_experiment/packages/lisk-transactions/src/transactions/3_vote_transaction.ts#L110

Reply:
For multisignature registration there will be a calculation involved. Also, would it be better to make it a function now in preparation for new fee system?

Reply:
maybe for fee, change it to `_fee: BigNum` and add `get fee(): BigNum` and `set fee(): BigNum` for now?